.github/workflows/gh-tools.sh

```diff
@@ -7,3 +7,8 @@ gh_export()
         shift;
     done
 }
+
+gh_path()
+{
+    echo "$1" >> $GITHUB_PATH
```

Comment:
Looking at the context, right now the environment setup is a dedicated step, does not run commands at all. I am not completely against it though, but I would wait for a concrete use, where this would make sense.